app/models/potential_match.rb

```diff
@@ -235,12 +246,19 @@ def self.progress(collection)
     collection_byline_key = byline_key(collection)
     unless REDIS_GENERAL.exists(collection_byline_key)
       score = 0
-      collection.signups.pseud_only.find_each do |pseud|
+      collection.signups.pseud_only.each do |pseud|
         REDIS_GENERAL.zadd collection_byline_key, score, pseud.byline
         score += 1
       end
     end
-    progress = (REDIS_GENERAL.zrank(collection_byline_key, current_byline)  * 100)/REDIS_GENERAL.zcount(collection_byline_key, 0, "+inf")
+    rank = REDIS_GENERAL.zrank(collection_byline_key, current_byline)
+    if rank.nil?
```

Comment:
This should normally not occur but it's there just so the page doesn't bomb out with a 500 error if for instance some user changes their pseud while matching is actually going (leading to them not being found in the previously generated list). The mod just sees a percentage of -1%.